sqlalchemy_filters/filters.py

```diff
@@ -58,6 +58,7 @@ class Operator(object):
         'not_in': lambda f, a: ~f.in_(a),
         'any': lambda f, a: f.any(a),
         'not_any': lambda f, a: func.not_(f.any(a)),
+        'find_in_set': lambda f, a: func.find_in_set(a, f),
```

Comment:
I think `in_set` is more consistent with other filter names

Reply:
👍